### DISCUSSION TITLE
Show dropdown menu on hover

### DISCUSSION THREAD

**Discussion by foremtehan** [tailwindlabs/tailwindcss#10983] [Help] [answered]
I'm newbie, How can i toggle dropdown menu on mouse hover?

https://play.tailwindcss.com/YFne3jzGYV

this `hover:[& ul]:visible` didn't work

**Comment by wongjn** [comment, marked as answer]
Spaces need to be escaped with an underscore, due to the fact that spaces are used to delimit class names.

You could also use `group` to avoid the need for an arbitrary variant. See both solutions at https://play.tailwindcss.com/NEItcvsn5K.
